holidays/countries/india.py

```diff
@@ -31,45 +31,83 @@ class India(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolida
     subdivisions = (
         "AN",  # Andaman and Nicobar Islands.
         "AP",  # Andhra Pradesh.
-        "AR",  # Arunāchal Pradesh.
```

Comment:
Offcial [ISO3166-2](https://www.iso.org/obp/ui/#iso:code:3166:IN) contains these names in the Indian System of Transliteration.

Reply:
Just my 2 cents here but we can probably include both this and ASCII-only version as aliases like what we've implemented for Italy a few months earlier.

Reply:
@ankushhKapoor, you can look at example with several aliases for subdiv in [Austria](https://github.com/vacanza/holidays/blob/62c60b4020f50f913e1ffca6e1b40f3eb4c93b1f/holidays/countries/austria.py#L36).